airbyte-integrations/connectors/source-pexels-api/source_pexels_api/spec.yaml

```diff
@@ -0,0 +1,71 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/pexels-api
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Pexel API Spec
+  type: object
+  required:
+    - api_key
+    - query
+    - photo_id
+    - video_id
+    - collection_id
```

Comment:
@btkcodedev I think this still not right. I don't want to retrieve only one photo and a video and a collection. I want to probably pass a collection id and retrieve all data inside of it. 
I think most valuable for this connector it is to only implement the endpoint for search or retrieve data from collections id.